en/development/configuration.rst

```diff
@@ -51,7 +51,7 @@ datasource
 persistent
     Whether or not to use a persistent connection to the database. When using
     SQLServer you should not enable persistent connections as it causes
-    difficult to diagnose crashes.
+    difficulty to diagnose crashes.
```

Comment:
if the sentence if misleading, you can update to `crashes difficult to diagnose` . I meant that the meaning of the sentence from my undertanding is that it causes `crashes` more than `difficulties`. I might be wrong I'm not a native speaker neither.